cloudevents/http/event.py

```diff
@@ -27,7 +27,7 @@ class CloudEvent:
     """
 
     def __init__(
-        self, attributes: typing.Dict[str, str], data: typing.Any = None
+            self, attributes: typing.Dict[str, str], data: typing.Any = None
```

Comment:
Can you please reformat the code using `black` utility? In particular, the indentation should be fixed by `black` here.

https://github.com/psf/black